src/ybus_calculations.jl

```diff
@@ -198,14 +267,16 @@ function _buildybus(
     branches,
```

Comment:
I am inclined to pass here a separate field with the three-winding transformers and handle them separately